### ISSUE TITLE
[1] Messages after socket disconnect

### ISSUE THREAD

**Issue by wmbutler** [bitshares/bitshares-ui#1380]
- [ ] Either stop checking the get_ticker Array or reconnect to the socket.

```
!!! GrapheneApi error:  get_ticker Array(2) Error: websocket state error:3
    at e.call (app.js:11)
    at e.exec (app.js:11)
    at app.js:11
    at t.h (app.js:11) {}
app.js:11 getMarketStats error: Error: websocket state error:3
    at e.call (app.js:11)
    at e.exec (app.js:11)
    at app.js:11
    at t.h (app.js:11)
app.js:11 !!! GrapheneApi error:  get_ticker Array(2) Error: websocket state error:3
    at e.call (app.js:11)
    at e.exec (app.js:11)
    at app.js:11
    at t.h (app.js:11) {}
app.js:11 getMarketStats error: Error: websocket state error:3
    at e.call (app.js:11)
    at e.exec (app.js:11)
    at app.js:11
    at t.h (app.js:11)
app.js:11 !!! GrapheneApi error:  get_ticker Array(2) Error: websocket state error:3
    at e.call (app.js:11)
    at e.exec (app.js:11)
    at app.js:11
    at t.h (app.js:11) {}
app.js:11 getMarketStats error: Error: websocket state error:3
    at e.call (app.js:11)
    at e.exec (app.js:11)
    at app.js:11
    at t.h (app.js:11)
app.js:11 !!! GrapheneApi error:  get_ticker Array(2) Error: websocket state error:3
    at e.call (app.js:11)
    at e.exec (app.js:11)
    at app.js:11
    at t.h (app.js:11) {}
app.js:11 getMarketStats error: Error: websocket state error:3
    at e.call (app.js:11)
    at e.exec (app.js:11)
    at app.js:11
    at t.h (app.js:11)
```

**Comment by svk31** [comment]
Isn't this a duplicate of #1346 ?

**Comment by wmbutler** [comment]
It's similar but these are different error messages.

**Comment by sschiessl-bcp** [comment]
If this is fixed already in bitsharesjs please close. If not, consider my fix #1434

Even if it's already fixed, it gives less code repetition.

**Comment by wmbutler** [comment]
We can close this as resolved but #1485 still stands.
